events/events.go

```diff
@@ -266,13 +291,23 @@ func ContainerMetric(msg *events.Envelope) Event {
 	return Event{
 		Fields: fields,
 		Msg:    "",
+		Type:   msg.GetEventType().String(),
 	}
 }
 
 func (e *Event) AnnotateWithAppData() {
+
 	cf_app_id := e.Fields["cf_app_id"]
-	if cf_app_id != nil && cf_app_id != "" {
-		appInfo := getAppInfo(fmt.Sprintf("%s", cf_app_id))
+	appGuid := ""
+	if cf_app_id != nil {
+		appGuid = fmt.Sprintf("%s", cf_app_id)
+		//fmt.Printf("Annotating for App Id %s\n",appGuid)
+	}
+
+	e.Fields["cf_origin"] = "newell-firehose"
+	e.Fields["cf_event_type"] = e.Type
```

Comment:
Why did you change event_type to cf_event_type, that will break dashboard or any logstash filter using event_type.